packages/server/postgres/helpers/OrgUserAudit.ts

```diff
@@ -0,0 +1,15 @@
+import {insertOrgUserAudit as insertQuery} from '../queries/generated/OrgUserAudit'
```

Comment:
if we change the sql file name to `insertOrgUserAuditRow` then we can import that without having to rename it up here